### PR TITLE
[16.0][MIG] product_variant_attribute_name_manager

### PR DESCRIPTION
**Pull request by JordiMForgeFlow**
supersedes https://github.com/OCA/product-attribute/pull/1236

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]
Functional review LGTM

File: product_variant_attribute_name_manager/views/product_variant_attribute_name_manager_view.xml [hunk 1]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" ?>
```
Comment (MiquelRForgeFlow) [inline product_variant_attribute_name_manager/views/product_variant_attribute_name_manager_view.xml]: why?

you should add `encoding="utf-8"` instead

**Review by yostashiro — APPROVED** [review]
Code review + functional test. 👍

**Comment by JordiMForgeFlow** [comment]
closing as https://github.com/OCA/product-attribute/pull/1236 was merged

**Comment by Nikul-OSI** [comment]
@JordiMForgeFlow If you did any extra functionality or code improvements then please create new enhancement PR.  :smile: 
Thank you

**Comment by JordiMForgeFlow** [comment]
Hi @Nikul-OSI 

Its not new improvements. The PR was missing features that were already part of V15. Now I opened a new PR with the missing FW https://github.com/OCA/product-attribute/pull/1465

**Pull request closed without merging**